vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -147,6 +152,12 @@ void FlyoutShadowNode::createView() {
       OnFlyoutClosed(*instance, m_tag, false);
   });
 
+  m_flyout.Opened([=](auto &&, auto &&) {
```

Comment:
Use explicit capture:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-this-capture